index.html

```diff
@@ -13219,16 +13219,20 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
 	</section>
 	<section id="tree_inclusion">
 	  <h2>Including Elements in the Accessibility Tree</h2>
-          <p>If not already excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
+          <p>If not excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Text nodes</li>
-            <li>Elements that may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>.</li>
-            <li>Elements that are focusable, or have an ID <a>attribute</a> and an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the implicit or explicit <a>semantics</a> of the required context role. In either case, the element may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+            <li>Elements that are not <a>hidden</a> and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>:
+              <ul>
+                <li>Elements that are focusable even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
+                <li>Elements that have an ID <a>attribute</a>, and are contained in or <a>owned</a> by an element that supports and has the <pref>aria-activedescendant</pref> property, and have an implicit or explicit role that matches the required context role of their containing element, and their containing element is either focusable or is controlled by an element that is focusable.</li>
```

Comment:
>Elements that have an ID attribute, and are contained in or owned by an element that supports and has the aria-activedescendant  property,

An option in a listbox controlled by a combobox does not meet this condition. The aria-activedescendant property is on the combobox, not on the owning listbox.

Looking at how we have worded the requirements for processing conflict resolution, I don't think we need all the details here. Instead, I suggest the following:

```suggestion
                <li>Elements that have an ID <a>attribute</a> and that have a role and context of a valid target of the <pref>aria-activedescendant</pref> property.</li>
```

Reply:
@cookiecrook You said in https://github.com/w3c/aria/pull/1100#discussion_r405919477 that this sentence was "much more clear". Matt has suggested that we reduce the sentence to its essential point, to "avoid duplicating the aria-activedescendant specification". Personally, I find the pared-down sentence easier to understand, but it would be great if you could take a quick look before I go ahead with the change.

I would reword it a tiny bit as follows:

> Elements that have an ID attribute, and have the role and context of a valid target of the aria-activedescendant property.

Reply:
@mcking65 Regarding "a role" vs. "the role", I think it's ok as "the role" because it has the role and context of "a" valid target, i.e. any of the possible valid targets.